net/rpmsg/rpmsg_sockif.c

```diff
@@ -782,7 +786,7 @@ static int rpmsg_socket_accept(FAR struct socket *psock,
         }
       else
         {
-          if (_SS_ISNONBLOCK(psock->s_flags))
+          if (_SS_ISNONBLOCK(conn->sconn.s_flags))
```

Comment:
Here in "else" block the conn variable is NULL.
_SS_ISNONBLOCK(conn->sconn.s_flags) results in crash.

Reply:
Reviewed, thanks.